bpmn-io/bpmn-js-differ
# feat: Add support for diffing changes to timer events. Added tests to verify

### Proposed Changes

closes #15 

The library has missed support for diffing changes made to `bpmn:TimerEventDefinition`. This PR adds support for this, by including `bpmn:TimerEventDefinition` in the list of tracked definitions. The solution is in whole taken from [this](https://github.com/bpmn-io/bpmn-js-differ/issues/15#issuecomment-1253710877) comment.

### Steps to test

1. Pull down repository
2. Run `npm install`
3. Run `npm run all`
4. See that all tests passes and that the results object from the timer event looks something like this:
```javascript
ChangeHandler {
  _layoutChanged: {},
  _changed: { TimerEventDefinition_0fgktse: { model: [Base], attrs: {} } },
  _removed: {},
  _added: {}
}
```

### Checklist

To ensure you provided everything we need to look at your PR:

* [x] **Brief textual description** of the changes present
* [x] **Visual demo** attached (hopefully the code sample is enough 😅 )
* [x] **Steps to try out** present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [x] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`
